### PR TITLE
Update nao_setup.rst: `sudo bin/changeField.py`

### PR DESCRIPTION
**Pull request by pzrq**
Add a Connecting to GameController Wifi section documenting how to use `sudo bin/changeField.py`.

### CODE REVIEW

File: source/setup/nao_setup.rst [hunk 1]
```diff
@@ -105,3 +105,33 @@ Finally, for all robots, run
...
+analagous to connecting to a wifi hotspot, one needs to specify things like the 
+`SSID <https://www.lifewire.com/definition-of-service-set-identifier-816547>`_:
+
+```
+    PC$ ssh nao@treebeard.local
+    # sudo bin/changeField.py <SSID>
+    sudo bin/changeField.py SPL_E
+```
+
+These could also have been set from the `runswift.cfg` file.
```
Comment (Ry-DS) [inline source/setup/nao_setup.rst]:
```suggestion
```
Having wifi related stuff in runswift.cfg when the runswift binary doesn't use it at all doesn't feel right. runswift.cfg should be for options used by the binary only

**Review by pzrq — COMMENTED** [review]
👍🏼

**Review by Ry-DS — APPROVED** [review]

**Comment by pzrq** [comment]
[Build](https://readthedocs.org/projects/runswift/builds/23622891/) is failing. Not sure why though, as `make html` works for me locally.

> Config validation error in build.os. Value build not found.

**Comment by pzrq** [comment]
Related: https://github.com/maartenbreddels/ipywebrtc/pull/140

**Comment by pzrq** [comment]
✅ Build fixed in e2c5907a14e77adc67aac5dd62efd4ca0900be55

**Pull request merged**